tests/steps/steps.py

```diff
@@ -32,7 +32,7 @@ def parse_string(text):
 algod_port = 60000
 kmd_port = 60001
 
-DEV_ACCOUNT_INITIAL_MICROALGOS: int = 10_000_000
+DEV_ACCOUNT_INITIAL_MICROALGOS: int = 100_000_000
```

Comment:
Why is this here too? Seems completely unrelated

Reply:
Tests have been v flakey with issues related to overspend errors. This change was an experiment to see if it fixed the overspend errors.

The tests passed on the first attempt once I included it, but I cant be sure this is _the_ fix

Reply:
I outlined the rationale for this one liner here as well: https://github.com/algorand/py-algorand-sdk/pull/459

It resolves current flaky tests because rekeying will change how transient accounts are funded in a non-deterministic way

Reply:
We already do this in the JS SDK but swept it under the rug in another PR: https://github.com/algorand/js-algorand-sdk/blob/develop/tests/cucumber/steps/steps.js#L122

Reply:
The rekey changes the ordering of the kmd wallets in py-algorand-sdk (I believe)
i think the change should be to provide non-deterministic access to the main network accounts instead